source/file/Exporter.js

```diff
@@ -60,6 +61,43 @@ class Exporter {
     }
   }
 
+  toSvg(tiles) {
```

Comment:
This method name is great, could you actually rename the above `fromTiles` to `toTopoJson` for consistency?